Phonebook.Frontend/src/app/app.component.ts

```diff
@@ -95,8 +99,19 @@ export class AppComponent implements OnInit {
     //   });
     // }
 
+    //if skip_cookies is set, dont show dialogs
+    if (this.skipDialog) {
```

Comment:
Can we add this directly to the if statements, which are opening the dialogs, below?

Reply:
What I mean is don't set the DisplayedNotificationVersion as is never displayed. Just don't call the functions opening the dialogs.

Reply:
I could do so, but the url parameter gets lost on state change, so on reload the dialogs will appear again if DisplayedNotificationVersion is not set.

Reply:
Can't quite follow. Normal usage of the app never reloads the page. The user has to use the saved favourites link in order to not get the dialog... But I get what you are saying.